### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by kianmeng**
Found via `codespell -S
po,doc,./misc/syntax,./src/vfs/extfs/helpers/README.it -L parm,rouge,sav,ect,vie,te,dum,clen,wee,dynamc,childs,ths,fo,nin,unx,nd,iif,iterm,ser,makrs,wil`

### CODE REVIEW

File: src/filemanager/file.c [hunk 1]
```diff
@@ -1516,7 +1516,7 @@ recursive_erase (file_op_total_context_t * tctx, file_op_context_t * ctx, const
   *          1 if there are no entries besides "." and ".." in the directory path points to,
   *          0 else.
   *
-  * ATTENTION! Be carefull when modifying this function (like commit 25e419ba0886f)!
+  * ATTENTION! Be careful when modifying this function (like commit 25e419ba0886f)!
   * Some implementations of readdir() in MC VFS (for example, vfs_s_readdir(), whuch is uded
```
Comment (zyv) [inline src/filemanager/file.c]:
```suggestion
  * Some implementations of readdir() in MC VFS (for example, vfs_s_readdir(), which is used
```

File: src/filemanager/panel.c [hunk 2]
```diff
@@ -4395,7 +4395,7 @@ panel_set_lwd (WPanel * panel, const vfs_path_t * vpath)
 /**
  * Creatie an empty panel with specified size.
  *
- * @param panel_name name of panel for setup retieving
```
Comment (zyv) [inline src/filemanager/panel.c:4398]: This and the following sounds weird. I guess the original was "receiving".

**Review by zyv — COMMENTED** [review]
@aborodin I've had a look and the changes look mostly fine, only one string is changed. Maybe you could take this into the cleanup branch?

P.S. `tests/src/viewer/viewertest.txt` - I guess this file was added by mistake...

**Comment by aborodin** [comment]
> Maybe you could take this into the cleanup branch?

Sure.

**Pull request closed without merging**